### PR TITLE
Replaced the broken link

### PR DESCRIPTION
**Pull request by J0SAL**
Updated the broken link in `Blockchain > Project Ideas > Article Format > Resource 1` with the new link from the same author

## Fixes Issue

#244 

## Updates
New Link: https://dev.to/edge-and-node/the-complete-guide-to-full-stack-web3-development-4g74

## Screenshots

image

### CODE REVIEW

**Review by siddhant-khisty — APPROVED** [review]
LGTM. Please wait until #243 is merged, as it changes some names. After the PR is merged and the conflicts are resolved, we can merge this.

**Comment by J0SAL** [comment]
@siddhant-khisty is #243 resolved yet?

**Comment by J0SAL** [comment]
@siddhant-khisty  Made the changes, could you please review it

**Pull request merged**
